src/components/docs-anchor/index.tsx

```diff
@@ -1,8 +1,7 @@
 import Link from 'next/link'
```

Comment:
🚧 I can't leave a comment on the specific line, but I think we only want to replace the `InlineLink` component reference here with the new `MSXInlineLink` component, and leave the other manual `<a>` elements as they are. `InlineLink` is the one that applies the custom styling for within docs content.

Reply:
The other `a` tags had custom styles being pass as well via the `s.anchor` class in `passableProps`

Reply:
I think we can totally make a follow-up task as you've asked in the other comment thread after these changes. 😊

Reply:
Do you mean after I swap `InlineLink` for `MdxInlineLink`?

Reply:
Yeah! So to summarize what I'm thinking should be done:

- Undo other changes in this file where `MdxInlineLink` is added
- Update `MdxInlineLink` to render `InlineLink`
- Replace [the `InlineLink` reference in this file](https://github.com/hashicorp/dev-portal/blob/main/src/components/docs-anchor/index.tsx#L37-L42) with `MdxInlineLink`

Reply:
Maybe I shouldn't have even made mdxInlineLink if we just want it to be a wrapper for inline link?